src/admin-views/tribe-options-upgrade.php

```diff
@@ -0,0 +1,121 @@
+<?php ob_start(); ?>
+
+<div id="modern-tribe-info" class="tribe-upgrade">
+	<?php
+	if ( tribe_events_views_v2_is_enabled() ) {
```

Comment:
I think if we had 3 separate templates, one top-level and two others for each conditional this would be more aligned with the Views V2 standards.
Also for conditionals inside of a template we use `<?php if () : ?>` and `<?php endif; ?>`

Reply:
this level of over-engineering is not something I can support

Reply:
Chunking up the views would work ok for `/views/`, but in `/admin-views/` it isn't necessary–and arguably less desirable.

Reply:
We do typically use `<?php if () : ?>` style conditionals in our templates, though. Not a requirement for merging, but worth knowing for the future.